### PR TITLE
[Feature] - library + no table details + page enhancements

### PR DESCRIPTION
**Pull request by jonycfu**
**PR checklist**

- [X ] Ensure you have added or ran the appropriate tests for your PR.
- [ X] DCO signed

**What type of PR is this?**
Feature
**What this PR does / why we need it**:
A mix of enhancements, including 
* sharing UI as library (rollup) via barrel `index.ts` exports
* refactored and merged many `CR` vs `SR` components (using `singleOnly` interface to differentiate `SR` vs `CR` logic)
* UI enhancements (see below list)

**Which issue(s) this PR fixes**:
28640, 28440, 28531

**Special notes for your reviewer**:
* Fix on mismatched schema columns: https://www.loom.com/share/89b7231555f240f4b4bae64cb9c73b52
* New Navigation (SR): https://www.loom.com/share/1fb1f343e3944e72b25e7c2c4b3953f1
* New Navigation (CR): https://www.loom.com/share/bac4124a76e844739389e542e10ffcee

**Does this PR introduce a user-facing change?**: Yes

* Removal of `TableDetails` page, and moving contents to the master/detail table column page
* Fix bug with schema list where mismatched columns was occurring
* Other UI enhancements (consistent icon, icon colors, tooltips)

### CODE REVIEW

File: static_report/rollup.config.mjs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import alias from '@rollup/plugin-alias';
+import dts from 'rollup-plugin-dts';
+import { terser } from 'rollup-plugin-terser';
+import peerDepsExternal from 'rollup-plugin-peer-deps-external';
+import nodePolyfills from 'rollup-plugin-node-polyfills';
+import { readFileSync } from 'fs';
+import path from 'path';
+
+const packageJson = JSON.parse(
+  readFileSync(new URL('./package.json', import.meta.url)),
```
Comment (neighborhood999) [inline static_report/rollup.config.mjs:13]: I remember you can import `package.json` directly:

```js
import pkg from './package.json';
```

So you don't need to use `readFileSync`.

File: static_report/package.json [hunk 2]
```diff
@@ -118,5 +148,8 @@
     "singleQuote": true,
     "trailingComma": "all"
   },
-  "packageManager": "yarn@3.2.1"
+  "packageManager": "yarn@3.2.1",
+  "volta": {
```
Comment (neighborhood999) [inline static_report/package.json:152]: What's this purpose?

File: static_report/rollup.config.mjs (hunk 1 above)
Comment (jonycfu) [reply static_report/rollup.config.mjs:13]: I thought so too, but i encountered `type: module` errors while doing that with a `*.mjs` file: 
Rollup docs: 
image

File: static_report/package.json (hunk 2 above)
Comment (jonycfu) [reply static_report/package.json:151]: I believe this was auto added by my node version manager, most likely it auto writes. I'll remove this.

https://docs.volta.sh/guide/

**Review by popcornylu — APPROVED** [review]
LGTM

Nice to have: The background and font of the table item in the table/column list. But it is not the blocker to merge the PR.

**Pull request merged**